src/app/util/attribute-storage.cpp

```diff
@@ -136,6 +137,7 @@ constexpr const EmberAfDeviceType fixedDeviceTypeList[]             = FIXED_DEVI
 DataVersion fixedEndpointDataVersions[ZAP_FIXED_ENDPOINT_DATA_VERSION_COUNT];
 
 AttributeAccessInterface * gAttributeAccessOverrides = nullptr;
+AttributeAccessInterfaceCache<1> gAttributeAccessInterfaceCache;
```

Comment:
nit: I would place a `kAttributeAccessInterfaceCacheSize` here to make it more visible as a set constant.

Reply:
Removed the template.